proto/def.bzl

```diff
@@ -74,9 +74,6 @@ def _go_proto_aspect_impl(_target, ctx):
         attr,
         include_deprecated_properties = False,
         importpath = attr.importpath,
-        importmap = attr.importmap,
```

Comment:
I would have expected logic such as `importmap = getattr(attr, "importmap", None)`. Is it clear that we don't need this?

Reply:
`go_context` calls `getattr` automatically if importmap etc are not passed: https://github.com/bazel-contrib/rules_go/blob/702d10674ec48a679488a725c86bf35121420698/go/private/context.bzl#L581